### PR TITLE
Extended the installation description

### PR DESCRIPTION
**Pull request by MHBalsmeier**
### Description
As mentioned in Issue 3949, installation of Iris on Ubuntu was very difficult. I described an easy way to install Iris on Ubuntu without conda in the installation file in the documentation.

### CODE REVIEW

**Comment by MHBalsmeier** [comment]
Just signed this CLA.

**Comment by MHBalsmeier** [comment]
Maybe someone can help me with why these tests fail?

**Comment by tkknight** [comment]
The two doc tasks have failed as there are some **warnings** in the build.  The build flag to **promote warnings to errors** is enabled, this ensures the documentation does not degrade over time.

I think the warnings seem to be: 

```
/tmp/cirrus-ci-build/docs/iris/src/developers_guide/contributing_code_formatting.rst:25: WARNING: undefined label: installing_from_source
/tmp/cirrus-ci-build/docs/iris/src/developers_guide/contributing_running_tests.rst:8: WARNING: undefined label: installing_from_source
/tmp/cirrus-ci-build/docs/iris/src/whatsnew/3.0.rst:262: WARNING: undefined label: installing_from_source

```
(see this on cirrus: https://cirrus-ci.com/task/6044147754729472?command=tests#L593)

I think you may have renamed the existing label **_installing_from_source** to be **_installing_from_source_with_conda**, meaning the existing references to the old label are now flagging an error.

I suggest you rename the label back, or correct the references in the above three files.

HTH.

**Comment by MHBalsmeier** [comment]
Don't know what went wrong now. Is it because of this:

(line  245) broken    https://www.unidata.ucar.edu/software/udunits/udunits-current/doc/udunits/udunits2.html - 404 Client Error: Not Found for url: https://www.unidata.ucar.edu/software/udunits/udunits-current/doc/udunits/udunits2.html

maybe?

**Comment by trexfeathers** [comment]
The change in #3942 is due to be merged back into `master` branch in the next couple of days, which will fix the link check failure. I suggest waiting for this mergeback and then rebasing your branch onto `master` once that happens.

**Comment by trexfeathers** [comment]
... alternatively you could just cherry pick the relevant commit ([83d1b72](https://github.com/SciTools/iris/commit/83d1b72442a22b5ad54a9cab8daa315e54061284)) onto your branch as soon as you like:

`git cherry-pick 83d1b72`

**Comment by tkknight** [comment]
The changes render html ok.  I would suggest that the new lines added are limited to **80 chars** in length as it makes reading the source **.rst** files much easier.   The only exceptions are when you embed commands, so the sudo and pip3 commands are ok on a single line.

I hope that makes sense.  We should be get to get this approved and then merged.

**Comment by MHBalsmeier** [comment]
The commit I did from the command line (c557a20) screwed it up I think. CLA-checker refuses to accept it now.

**Comment by trexfeathers** [comment]
>I am not sure if I can fix this now.

Re-do those commits using a `rebase` or `reset`, then `push -f` to apply those changes to this PR?

**Comment by MHBalsmeier** [comment]
Should be okay now.

**Comment by bjlittle** [comment]
@MHBalsmeier Awesome! Thanks for taking the time to contribute this PR - it's really appreciated 😀

As an aside, I'm soon to push the `iris` 3.0.0 release to PyPI. It would be great if you could test a `pip` install given your installation on `ubuntu`.

Fancy confirming whether the install of 3.0.0 works from PyPI?

**Comment by bjlittle** [comment]
@MHBalsmeier Also, could you make an entry for your PR in the `docs/iris/src/whatsnew/latest.rst`?

We're keen to recognise your valued contribution 👍

File: docs/iris/src/installing.rst [hunk 1]
```diff
@@ -41,11 +41,45 @@ need the Iris sample data. This can also be installed using conda::
...
+before and after installation of Debian packages.
+
+The rest can be done with pip. Begin with numpy::
+
+  pip3 install numpy
+
+Finally, Iris and its Python dependencies can be installed with the following
+command::
+
+  pip3 install astropy tqdm oktopus bs4 setuptools nc_time_axis cython cfunits cf-units cf-python pandas stratify pyugrid scitools-pyke black imagehash nose asv scitools-iris
```
Comment (bjlittle) [inline docs/iris/src/installing.rst:72]: @MHBalsmeier You seem to have quite a few extra dependencies and optional dependencies mixed in here... would it be possible to strip this example back to it's essential bare minimum?

I'm guessing that this is the package dependency requirements for your specific area of research/work.

The following appear to extra dependencies for your particular needs:
* `astropy`
* `tqdm`
* `oktopus`
* `bs4`
* `cython`
* `cf-python`
* `asv`

The following appear to be optional dependencies of `iris`, but not strictly necessary:
* `nc_time_axis`
* `pandas`
* `stratify`
* `pyugrid`
* `black`
* `imagehash`
* `nose`

I'm also assuming that `cfunits` isn't a necessary dependency for `cf-units`? (`cfunits` is a similar package as our SciTools `cf-units`, which is maintained by David Hassell from Reading University)

So, I guess my question is, could this actually boil down to simply:
```shell
> pip3 install setuptools cf-units scitools-pyke scitools-iris
```
Although, I'm guessing (off the top of my head) that `cython` is necessary for `cf-units`? If that's the case then we have:
```shell
> pip3 install setuptools cython cf-units scitools-pyke scitools-iris
```
Is it possible for you to confirm this?

**Comment by bjlittle** [comment]
@MHBalsmeier BTW I've just uploaded `scitools-iris` `3.0.0` to PyPI... fancy giving it a try? :smile:

**Comment by MHBalsmeier** [comment]
The way I have it now works with v3.0.0. cftime would throw an error if not included explicitly with a version number due to a dependency conflict.

**Review by bjlittle — APPROVED** [review]
@MHBalsmeier Awesome work! Thank you! :100: :tada: :beers: 

LGTM... @trexfeathers I'll leave you review and merge when your happy :+1:

**Pull request merged**